lib/LaTeXML/Package/pgf.sty.ltxml

```diff
@@ -38,9 +38,9 @@ DefPrimitive('\lxSVG@set@color', sub {
     MergeFont(color => LookupValue('color_pgfstrokecolor')); });
 
 # This makes this conditional always true;  Why???
-DefConditionalI('\ifpgf@relevantforpicturesize', undef, sub { 1; });
-DefMacroI('\pgf@relevantforpicturesizefalse', undef, '');
-DefMacroI('\pgf@relevantforpicturesizetrue',  undef, '');
+# DefConditionalI('\ifpgf@relevantforpicturesize', undef, sub { 1; });
+# DefMacroI('\pgf@relevantforpicturesizefalse', undef, '');
+# DefMacroI('\pgf@relevantforpicturesizetrue',  undef, '');
```

Comment:
Cool! You might as well just remove those lines, as well as the obsolete comment preceding them.